### ISSUE TITLE
Turning off global search

### ISSUE THREAD

**Issue by ashramsey** [avo-hq/avo#614]
Hi, please redirect me if this is not the place to ask questions. 

I recently purchased the pro license so I could use pundit policies.

The admin area now has a global search.

Is there a way to turn this feature off? I didn't find anything in the docs.

Thanks!

**Comment by ashramsey** [comment]
for now, I've just added my own global_search partial. I see theres a `ProLicense` abilities list, is there a way to edit this list from my app codebase?

**Comment by adrianthedev** [comment]
Hey @ashramsey. Not at the moment but will add that ability today for sure. 
Thanks for reaching out to us with this issue.

**Comment by adrianthedev** [comment]
Hey @ashramsey. We just merged a PR that helps with this issue. You can hide the global search using the following setting:

```ruby
# config/initializers/avo.rb
Avo.configure do |config|
  config.disabled_features = [:global_search]
end
```

More on this in the [docs](https://docs.avohq.io/1.0/search.html#hide-the-global-search).

I'll cut a release soon with this change.

**Comment by npickelny** [comment]
Hey @ashramsey. I have a question, were you able to set up the Pundit policies correctly?? Im having a hard time trying to make them work

**Comment by ashramsey** [comment]
> Hey @ashramsey. We just merged a PR that helps with this issue. You can hide the global search using the following setting:
> 
> ```ruby
> # config/initializers/avo.rb
> Avo.configure do |config|
>   config.disabled_features = [:global_search]
> end
> ```
> 
> More on this in the [docs](https://docs.avohq.io/1.0/search.html#hide-the-global-search).
> 
> I'll cut a release soon with this change.

Thats great, Ill give it a go. Thanks for the quick turnaround!

**Comment by ashramsey** [comment]
> Hey @ashramsey. I have a question, were you able to set up the Pundit policies correctly?? Im having a hard time trying to make them work

@npickelny Yes, I had no issue setting them up as per the docs. Running the `bin/rails g pundit:install` first adds the `application_policy.rb` which sets all crud actions to `false`, then I added the policy file for my resource eg `bin/rails g pundit:policy Job` which inherits off the `ApplicationPolicy` and override the  actions as I needed.

I don't think there was anything else needed to be done.

Let me know if you want me to share those 2 policies.

**Comment by adrianthedev** [comment]
@npickelny can you check if you set up the `current_user_method`? This is how Avo knows who your current user is and does the authorization against.

We improved the docs this morning.

Here's a docs page on it.
https://docs.avohq.io/1.0/authentication.html#customize-the-current-user-method
